### PR TITLE
removing from financial/scientific

### PR DESCRIPTION
**Pull request by jdamiba**

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
this includes both financial and scientific but OK, i'll review them together :)

File: _posts/plotly_js/scientific/carpet/2017-05-16-set_coordinates.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 name: Set X and Y Coordinates
-<!-- plot_url: https://codepen.io/plotly/embed/eWLKbJ/?height=500&theme-id=15263&default-tab=result -->
```
Comment (nicolaskruchten) [inline _posts/plotly_js/scientific/carpet/2017-05-16-set_coordinates.html]: this diff looks off

**Comment by nicolaskruchten** [comment]
💃

**Comment by nicolaskruchten** [comment]
(well, except for that one comment)

File: _posts/plotly_js/scientific/carpet/2017-05-16-set_coordinates.html (hunk 1 above)
Comment (jdamiba) [reply _posts/plotly_js/scientific/carpet/2017-05-16-set_coordinates.html]: Fixed typo.

**Pull request merged**